scripts/ci/deploy/config/databend-query-node-1.toml

```diff
@@ -110,3 +103,45 @@ data_path = "./.databend/stateless_test_data"
 # endpoint_url = "<your-endpoint>"
 # access_key_id = "<your-key-id>"
 # access_key_secret = "<your-account-key>"
+
+
+[cache]
+
+### table meta caches ###
+# Enable table meta cache. Default is true.
+# Set it to false wll disable all the table meta caches
+enable_table_meta_caches = true
```

Comment:
enable_table_meta_caches -> enable_table_meta_cache？

Reply:
Prefer to table_meta_cache_ebabled, this will same style with table_index_bloom_enabled